executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -197,11 +218,31 @@ impl<'a, T: FieldElement> FixedLookup<'a, T> {
                 .then_some((i.id, i))
             })
             .collect();
+
+        let degree = fixed_data
+            .fixed_cols
+            .values()
+            .map(|col| col.values_max_size().len())
+            .max()
+            // No fixed column, so this FixedLookup instance does nothing
+            .unwrap_or(0) as u64;
+
+        // This currenlty just takes one element with the correct name
+        // When we support more than one element, we need to have a vector of logup_multiplicity_columns: Vec<Option<PolyId>>
+        let logup_multiplicity_column: Option<PolyID> = fixed_data
```

Comment:
Yes, but figuring out that mapping is not straight-forward (would need to find and analyze the polynomial identities belonging to the lookup), whereas it's trivial after #1378 (because we'll have an explicit annotation). So, in my opinion, we should get that done first.